rr_viz/src/rr_viz/widgets/mission_dashboard/mapping/slam_supervisor.py

```diff
@@ -110,12 +129,15 @@ def switchToLocalizationSlot(self):
 
     def runLocalization(self):
         _str = StringRequest()
-        _str.str = self.mapListWidget.currentItem().text().split(".")[0]
+        _str.str = self.mapListWidget.currentItem().text().split(".")[0].strip()
```

Comment:
I think this strip might be causing an issue I'm about to describe (just a gut feel, the issue might be somewhere else actually).